indexing-service/src/main/resources/indexer_static/js/console-0.0.1.js

```diff
@@ -2,39 +2,56 @@
 
 var oTable = [];
 
+var killTask = function(taskId) {
+  if(confirm('Do you really want to kill: '+taskId)) {
+    $.ajax({
+      type:'POST',
+      url: '/druid/indexer/v1/task/'+ taskId +'/shutdown',
+      data: ''
+    }).done(function(data) {
+      alert('Submitted kill request for: '+data.task);
```

Comment:
qn - do we need an alert here? 

i don't remember this but once the task is killed, it will show up in Complete task with KILLED status immediately right?